src/tests/ftest/launch.py

```diff
@@ -759,7 +758,7 @@ def get_test_files(test_list, args, yaml_dir):
         # Display the modified yaml file variants with debug
         command = ["avocado", "variants", "--mux-yaml", test_file["yaml"]]
         if args.extra_yaml:
-            command.append(args.extra_yaml)
+            command.extend(args.extra_yaml)
```

Comment:
Do we need the conditional before this?

Reply:
Yes. `ArgumentParser` sets the default as `None`, which is not iterable. Even when I changed the default to `[]`, `ArgumentParser` set it to `None`